deploy/kube-config/influxdb/grafana.yaml

```diff
@@ -23,7 +23,7 @@ spec:
         env:
         - name: INFLUXDB_HOST
           value: monitoring-influxdb
-        - name: GRAFANA_PORT
+        - name: GF_SERVER_HTTP_PORT
```

Comment:
Potential issue renaming the environment variable here, since the image version hasn't bumped yet. Grafana should be configured correctly, but `setup_grafana` won't recognize the new environment variable on older versions.